app/src/main/java/com/vci/vectorcamapp/surveillance_form/presentation/SurveillanceFormScreen.kt

```diff
@@ -1,236 +1,399 @@
-import androidx.compose.foundation.rememberScrollState
-import androidx.compose.foundation.verticalScroll
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.ArrowBack
 import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
 import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.Scaffold
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.text.input.KeyboardType
 import androidx.compose.ui.tooling.preview.PreviewLightDark
 import androidx.compose.ui.unit.dp
-import com.vci.vectorcamapp.core.domain.util.onError
-import com.vci.vectorcamapp.core.domain.util.onSuccess
-import com.vci.vectorcamapp.surveillance_form.location.data.toString
 import com.vci.vectorcamapp.surveillance_form.domain.enums.CollectionMethodOption
 import com.vci.vectorcamapp.surveillance_form.domain.enums.DistrictOption
 import com.vci.vectorcamapp.surveillance_form.domain.enums.LlinBrandOption
 import com.vci.vectorcamapp.surveillance_form.domain.enums.LlinTypeOption
 import com.vci.vectorcamapp.surveillance_form.domain.enums.SentinelSiteOption
 import com.vci.vectorcamapp.surveillance_form.domain.enums.SpecimenConditionOption
 import com.vci.vectorcamapp.surveillance_form.location.data.LocationError
+import com.vci.vectorcamapp.surveillance_form.location.data.toString
 import com.vci.vectorcamapp.surveillance_form.presentation.components.DatePickerField
 import com.vci.vectorcamapp.surveillance_form.presentation.components.DropdownField
 import com.vci.vectorcamapp.surveillance_form.presentation.components.TextEntryField
 import com.vci.vectorcamapp.surveillance_form.presentation.components.ToggleField
+import com.vci.vectorcamapp.ui.extensions.customShadow
+import com.vci.vectorcamapp.ui.extensions.dimensions
+import com.vci.vectorcamapp.ui.theme.LocalColors
 import com.vci.vectorcamapp.ui.theme.VectorcamappTheme
+import com.vci.vectorcamapp.ui.theme.screenHeightFraction
 
 @Composable
 fun SurveillanceFormScreen(
     state: SurveillanceFormState,
     onAction: (SurveillanceFormAction) -> Unit,
     modifier: Modifier = Modifier
 ) {
-    val context = LocalContext.current
+    val pageHeaderHeight = 0.25f
+    val pageBodyOffset = 0.2f
 
-    val verticalScrollState = rememberScrollState()
+    Scaffold (
+        modifier = modifier
```

Comment:
Dont need a scaffold. It is intended to be rendered only once, and is rendered in the NavGraph for each screen.